### ISSUE TITLE
OSX Sequoia causing OpenEmu to crash?

### ISSUE THREAD

**Issue by evilaj2010** [OpenEmu/OpenEmu#5046]
So I've been using OpenEmu all this year and it was working just fine but as soon as I installed the OSX Sequoia, it can no longer start the app. Is anyone else having this same problem?

**Comment by benjamin-smith-it** [comment]
> So I've been using OpenEmu all this year and it was working just fine but as soon as I installed the OSX Sequoia, it can no longer start the app. Is anyone else having this same problem?

Yes. I've also been having this exact issue. Shame because the app was bullet proof before the upgrade.

**Comment by yourgluestick** [comment]
I actually noticed that in my case the reason of crashing is a gamepad being plugged in. After being unplugged it seems to cure itself. Tried on a couple of GBA and NDS ROMs (Castlevania Aria of Sorrow & Portrait of Ruin in particular) 

Sequoia 15.0 
MacBook Air M3

**Comment by cookies92** [comment]
> I actually noticed that in my case the reason of crashing is a gamepad being plugged in. After being unplugged it seems to cure itself. Tried on a couple of GBA and NDS ROMs (Castlevania Aria of Sorrow & Portrait of Ruin in particular)
> 
> Sequoia 15.0 MacBook Air M3

Yea, it's the same for me as well, controllers seem to make it crash on start up. And when I open it, then plug in a controller and try to play a game, the core unexpectedly quits. OpenEmu is literally unusable for me since the update. I use an Xbox Controller through a USB to provide power but it seems to be connected by bluetooth.

15.0.1 on a M1 Mac Mini.

Okay, now that I put batteries into my controller, now everything works and OpenEmu doesn't crash. It's really weird. It's something more than OpenEmu, at least for me it seems like USB connected controllers do not work anymore on macOS. Even Steam games didn't work. Just OS stuff was working but no games worked.
